modules/utility/init.zsh

```diff
@@ -118,7 +118,9 @@ alias ll='ls -lh'        # Lists human readable sizes.
 alias lr='ll -R'         # Lists human readable sizes, recursively.
 alias la='ll -A'         # Lists human readable sizes, hidden files.
 alias lm='la | "$PAGER"' # Lists human readable sizes, hidden files through pager.
-alias lx='ll -XB'        # Lists sorted by extension (GNU only).
+if [[ "$OSTYPE" != darwin* ]]; then
```

Comment:
Can we move this to the top? Makes the code more maintainable in the long run. (this can be considered as a nitpick :) )

Reply:
can do. But I just considered to keep changes at a minimum. 

But regarding readability I also had something in mind, and that is why I did #1793 with helpers. Presuming that helper would exist, would that calm your nit'ish feeling a little? :wink:

Reply:
I don't have a view on #1793, so I was waiting for the other contributors decide. Probably should add @belak to the reviewers list :smile: for #1793

Reply:
#1793 is highly optional. I was just mentioning it, because I interpreted your comment as a readability one.

Will add belak then... thx for the hint 😄

Reply:
My comment is totality a readability one :smile:

Reply:
@srijanshetty 

I changed this code now. Now I check for the capability of ls instead of the OS running. 

Just the fact that you're running on darwin doesn't prove your having a BSD style `ls`.
Since I'm aware of its existence I myself have the gnu coreutils installed here, so for myself this fix is not needed any more. :wink: